man/swaync.5.scd

```diff
@@ -549,7 +563,13 @@ config file to be able to detect config errors
 		},
 		"mpris": {
 			"image-size": 96,
-			"image-radius": 12
+			"image-radius": 12,
+			"blacklist": {
+                "firefox": {
+                    "state": "blacklisted",
```

Comment:
What states can be used here? If only "blacklisted", we should remove that due to it already being in a object called "blacklist"